src/pybind/mgr/snap_schedule/fs/schedule.py

```diff
@@ -30,20 +30,20 @@
     log.info(('Couldn\'t find datetime.fromisoformat, falling back to '
               f'static timestamp parsing ({SNAP_DB_TS_FORMAT}'))
 
-    def ts_parser(ts):
+    def ts_parser(data_string: str) -> datetime: # type: ignore
```

Comment:
@sebastian-philipp otherwise, i'd have:
```
snap_schedule/module.py:10: note: ... from here:
snap_schedule/fs/schedule.py: note: In function "ts_parser":
snap_schedule/fs/schedule.py:33: error: Incompatible redefinition (redefinition with type "Callable[[Arg(str, 'data_string')], datetime]", original type "Callable[[Arg(str, 'date_string')], datetime]")
Found 1 error in 1 file (checked 1 source file)
```

Reply:
sigh. fair enough